src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -205,8 +206,29 @@ const AppWrapper = (props: { children: ReactNode }) => {
       // Agent.agent.credentials.syncACDCs(),
       // ]);
     };
+    const removePendingDeletion = async () => {
+      const pendingDeletions =
+        await Agent.agent.connections.getConnectionsPendingDeletion();
+
+      for (const id of pendingDeletions) {
+        await Agent.agent.connections.deleteConnectionById(id);
+        dispatch(removeConnectionCache(id));
+      }
+    };
+
+    const reconnectPendingConnection = async () => {
+      const pendingDeletions =
```

Comment:
Please name the function `resolvePendingConnections` and this variable `pendingConnections`. We aren't deleting in this function!